packages/mui-material/src/Grid/Grid.test.js

```diff
@@ -746,7 +746,7 @@ describe('Material UI <Grid />', () => {
         generateRowGap({
           ownerState: {
             container: true,
-            rowSpacing: { xs: 1, sm: 2 },
+            rowSpacing: { xs: 1, sm: '16px' },
```

Comment:
Shouldn't we keep the `{ sm: 2 }` test as well instead of replacing it?

Reply:
Added `md: '16px'` instead.